script.js

```diff
@@ -257,3 +257,12 @@ cancelbtn.forEach((button) => {
     close3(popup3);
   });
 });
+
+const email = document.getElementById('email');
+email.addEventListener('input', () => {
+  if (email.validity.patternMismatch) {
+    email.setCustomValidity('Form submission error! Please type a valid email address in lowercase!');
+  } else {
+    email.setCustomValidity('');
```

Comment:
- [ ] Nice work implementing the validation :clap: However even when all letters in the email field are lowercase the error message still prevent the form to be submit. Kindly check it and make sure the error appears only when the email entered contain a capital letter to follow the project requirement.

Reply:
- Modified the pattern `regexp` in `index.HTML` file form email input element to conform to project requirements.